src/store/slices/blogsSlice.js

```diff
@@ -50,126 +31,14 @@ const blogsSlice = createSlice({
-            })
-        },
-        addCommentLike(state,action){
-            // assumption:
-            // action.payload === {blogId, commentId}
-            const {
-                blogId,
-                commentId
-            } = action.payload;
-        
-            state.map((blog)=>{
-                if(blog.id === blogId){
-                    blog.comments.map((comment)=>{
-                        if(comment.id === commentId){
-                            comment.upvotes++;
-                        }
-                    })
-                }
-            })
-        },
-        addCommentDislike(state,action){
-            // assumption:
-            // action.payload === {blogId, commentId}
-            const {
-                blogId,
-                commentId
-            } = action.payload;
-            
-            state.map((blog)=>{
-                if(blog.id === blogId){
-                    blog.comments.map((comment)=>{
-                        if(comment.id === commentId){
-                            comment.downvotes++;
-                        }
-                    })
-                }
-            })
-        },
-        addComment(state,action){
-            // assumption:
-            // action.payload === {blogId, {comment object}}
-            const {
-                blogId,
-                commentObject
-            } = action.payload;
-        
-            state.map((blog)=>{
-                if(blog.id === blogId){
-                    blog.comments.push(commentObject);
-                }
-            })
-        },
-        removeComment(state,action){
-            // assumption:
-            // action.payload === {blogid, commentId}
-            const {
-                blogId,
-                commentId
-            } = action.payload;
-        
-            state.map((blog)=>{
-                if(blog.id === blogId){
-                    const updated = blog.comments.filter((comment)=>{
-                        return comment.id !== commentId;
-                    })
-                    blog.comments = updated;
-                }
-            });
-        
-        },
-        updateComment(state,action){
-            // assumption: 
-            // action.payload === {blogId, commentId, commentText}
-            const {
-                blogId,
-                commentId,
-                commentText
-            } = action.payload;
-        
-            state.map((blog)=>{
-                if(blog.id === blogId){
-                    blog.comments.map((comment)=>{
-                        if(comment.id === commentId){
-                            comment.comment = commentText;
-                        }
-                    })
-                }
-            })
         }
     }
 });
 
 export const{
-    addDislike,
-    addLike,
     updateBlog,
     removeBlog,
     addBlog,
-    addCommentLike,
-    addCommentDislike,
-    addComment,
-    removeComment,
-    updateComment
```

Comment:
This file has SUCCESS written all over it, so much improvement from last version :)